### ISSUE TITLE
Scan in rest discovery returns 404

### ISSUE THREAD

**Issue by mathiasimmer** [openhab/openhab2#227]
So I was digging around trying to find out why no devices was found when I tried to scan for new devices. 
The output of http://localhost:8080/rest/discovery is:
```
[
"yahooweather",
"ipp",
"pulseaudio",
"max",
"sonos",
"wmbus"
]
```
But when http://localhost:8080/rest/discovery/scan/wmbus is being called (by the Paper UI) it responds with a 404 hence the discovery implementation is never called.

**Comment by kaikreuzer** [comment]
This URL only supports POST, see https://github.com/eclipse/smarthome/blob/master/bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/discovery/DiscoveryResource.java#L61
(sorry, REST API documentation is still on the todo list...)

**Comment by mathiasimmer** [comment]
Paper UI submits a POST request however it still returns 404:

![udklip2](https://cloud.githubusercontent.com/assets/8802601/7470979/f186352a-f321-11e4-9eb2-a50441469822.JPG)

**Comment by mathiasimmer** [comment]
Also there seems to be a mismatch in the urls: PaperUI expects /discovery/scan/{bindingId} where as the declared is /discovery/{bindingId}/scan but neither works (tried with POST).

**Comment by mathiasimmer** [comment]
Actually the declared url seems to be /discovery/bindings/{biidningId}/scan - So appears to be a bug in PaperUI

**Comment by cdjackson** [comment]
Didn’t this endpoint address change recently (i.e. in the past week) so maybe the paper UI hasn’t been updated?

**Comment by kaikreuzer** [comment]
Yes it has changed and yes, the Paper UI has been updated. I guess @mathiasimmer might still have an old version of it?

**Comment by mathiasimmer** [comment]
Might be - Where can I find the latest ? The one I got is from https://github.com/openhab/openhab2/blob/master/docs/sources/development/ide.md#paper-ui

**Comment by kaikreuzer** [comment]
You now don't need to install it manually anymore - I have just changed the target platform, so that it is automatically picked up: https://github.com/openhab/openhab2/commit/e31566d9f4581e506b96f7fabd4d819a07a7bd6e
In my tests, the discovery triggering works flawlessly with this setup.

**Comment by mathiasimmer** [comment]
Sounds good! I got it to work. I must admit that it probably was my own fault. I found it a tad confusing with the version numbers: the old was 2.0.0-SNAPSHOT, the new was 0.8.0 but when running the UI, it said 0.6 down in the left corner of the interface.
At the moment I'm not able to confirm whether your latest update works for me because it appears that the target location definitions are offline; at least I cannot access http://dist.springsource.org/release/GRECLIPSE/e4.4/ etc. 
but I don't think that is related to your update.
